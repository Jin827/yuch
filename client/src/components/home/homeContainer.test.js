import React from 'react';
import { shallow } from 'enzyme';
import HomeContainer from './homeContainer';

describe('<HomeContainer />', () => {
  const wrapper = shallow(<HomeContainer />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
