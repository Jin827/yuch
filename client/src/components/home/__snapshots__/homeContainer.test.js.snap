// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeContainer /> renders correctly 1`] = `
Array [
  <Header
    key="1"
  />,
  <HomeMain
    key="2"
  />,
  <WithStyles(Connect(ReserveContainer))
    key="3"
  />,
  <Board
    key="4"
  />,
]
`;
